.github/workflows/benchmark-tests.yml

```diff
@@ -0,0 +1,68 @@
+name: Benchmark Tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 10 * * 1'
+
+env:
+  # Enable versioned runner quiet mode to make CI output easier to read:
+  OUTPUT_MODE: quiet
+
+jobs:
+  should_run:
```

Comment:
do we need this? since you removed running on push/PR this seems superfluous.

Reply:
I can take that one out as well.